nest-forms-backend/src/utils/middlewares/logger.service.ts

```diff
@@ -7,13 +7,15 @@ import { LineLoggerSubservice } from '../subservices/line-logger.subservice'
 @Injectable()
 export default class AppLoggerMiddleware implements NestMiddleware {
   use(request: Request, response: Response, next: NextFunction): void {
+    response.set('middlewareUsed', 'true')
     const { method, originalUrl, body, ip, userAgent, userId } =
       this.extractRequestData(request)
     const startAt = process.hrtime()
     const { statusMessage, statusCode } = response
 
     const { send } = response
     response.send = (exitData: string | object | Buffer | Array<any>) => {
+      response.set('middlewareUsed', undefined)
```

Comment:
response.set set's [response headers](https://expressjs.com/en/api.html#res.set). While usable, it does not seem like the best place to store data for application.

there is [res.locals](https://expressjs.com/en/api.html#res.locals) which looks better suited - what about moving it there ?